website/client/src/components/settings/resetModal.vue

```diff
@@ -38,6 +38,7 @@ export default {
     },
     async reset () {
       const response = await axios.post('/api/v4/user/reset');
+      window.location.reload(true);
```

Comment:
Hmm, I was a bit confused why this was necessary, I thought the reset already had a built in reload... It looks like it didn't although it was necessary, thanks!

Could you remove the extra lines in this method? since they're not used anymore I think it can be simplified to:

```
async reset () {
      await axios.post('/api/v4/user/reset');
      window.location.reload(true);
}
```

Reply:
Not sure what happend here...  
I cannot read much out of the lint error report. Shall I better put the lines of code back?